recipes/FC-Virus/mete.yaml

```
@@ -0,0 +1,25 @@
+    version: "1.0.0"
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - make
+  host:
+    - boost
+  run:
+    - boost
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)